docs/resources/dynamic_table.md

```diff
@@ -42,15 +42,16 @@ resource "snowflake_dynamic_table" "dt" {
 ### Optional
 
 - `comment` (String) Specifies a comment for the dynamic table.
+- `initialize` (String) Initialize trigger for the dynamic table. Can only be set on creation.
```

Comment:
as in refresh_mode specify what are the allowed values

Reply:
Done